src/profiling/power_meter.cpp

```diff
@@ -0,0 +1,48 @@
+#include <string>
+#include <vector>
+
+#include <util/config.hpp>
+
+#include "power_meter.hpp"
+
+namespace nest {
+namespace mc {
+namespace util {
+
+class power_meter: public meter {
+    std::vector<energy_size_type> readings_;
+
+public:
+    std::string name() override {
+        return "energy";
+    }
+
+    std::string units() override {
+        return "J";
+    }
+
+    std::vector<double> measurements() override {
+        std::vector<double> diffs;
+
+        for (auto i=1ul; i<readings_.size(); ++i) {
```

Comment:
That `std::size_t` is no larger than an unsigned long is an unwarranted assumption (e.g. unsigned long is only 4 bytes under windows.) Why not just use `std::size_t`?